### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by tgallagher2017**
Add more documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,9 +58,13 @@ Most of these are for templating in 11ty. See the 11ty docs for more info. If yo
 
 This directory holds your Lit components. If you add a component here, to hook it up to SSR, add it to the `src/ssr.ts` file and make sure to add it to `tsEntrypoints` in `esbuild.config.mjs` to make sure it's included in the bundler. If you don't want to SSR a component, you can add it to the appropriate file in `src/pages/<your page name>.ts`;
 
+#### `site/hydration-entrypoints`
+
+This holds files that include 'import' notation to import the components used in server side rendering (SSR) and require hydration. The pages with .ts extention match the structure and name as those files in the `site` folder.
```
Comment (e111077) [inline README.md]: Wording is a bit cloudy here. Can you include a code block example please?

**Review by e111077 — CHANGES_REQUESTED** [review]
Thanks so much for the submission! Just one small request

**Comment by tgallagher2017** [comment]
Hi,

Sorry for the delay in updating the README. I was busy finishing writing my
novel. Now I'm back to writing software.

Tim

**Review by e111077 — APPROVED** [review]
Congrats on the novel!

**Pull request merged**